scripts/poll_running_amplify_job.sh

```diff
@@ -5,6 +5,8 @@ set -euo pipefail
 app_id=$1
 branch_name=$2
 
+sleep 15
+
 job_id=$( aws amplify list-jobs --app-id $app_id --branch-name $branch_name | jq -r '.jobSummaries[0].jobId' )
```

Comment:
Can this be a poll rather than a fixed 15 second sleep?  Maybe there is a nice way to use the `until` bash command?

Reply:
Further down we have a 10s sleep, can this be lowered or will we hit AWS throttling?

Reply:
updated